posthog/api/paths.py

```diff
@@ -85,7 +85,7 @@ def _apply_start_point(self, start_comparator: str, query_string: str, start_poi
     def _add_elements(self, query_string: str) -> str:
         element = 'SELECT \'<\'|| e."tag_name" || \'> \'  || e."text" as tag_name_source, e."text" as text_source FROM "posthog_element" e JOIN \
                     ( SELECT group_id, MIN("posthog_element"."order") as minOrder FROM "posthog_element" GROUP BY group_id) e2 ON e.order = e2.minOrder AND e.group_id = e2.group_id where e.group_id = v2.group_id'
-        element_group = 'SELECT g."id" as group_id FROM "posthog_elementgroup" g where v1."elements_hash" = g."hash"'
+        element_group = 'SELECT g."id" as group_id FROM "posthog_elementgroup" g where v1."elements_hash" = g."hash_id"'
```

Comment:
There's two more places where `hash` needs to be replaced. On line 61 in the select and the join condition in the back

Reply:
Done, also added a test.